buildSrc/src/main/java/com/osacky/flank/gradle/YamlWriter.kt

```diff
@@ -44,6 +44,7 @@ internal class YamlWriter {
     val smartFlankDisableUpload = config.smartFlankDisableUpload
     val localResultsDir = config.localResultsDir.orNull
     val testTargetsAlwaysRun = config.testTargetsAlwaysRun
+    val outputStyle = config.outputStyle.orNull
```

Comment:
i think `orNull` isn't necessary or has no effect here since the convention is set.